hack/docgen.py

```diff
@@ -0,0 +1,239 @@
+import json
```

Comment:
Please do not introduce Python unless you're sure the rest of the team are proficient.